ivy/functional/backends/jax/activations.py

```diff
@@ -37,7 +37,7 @@ def sigmoid(x: JaxArray, /, *, out: Optional[JaxArray] = None) -> JaxArray:
 
 
 def softmax(
-    x: JaxArray, /, *, axis: Optional[int] = None, out: Optional[JaxArray] = None
+    x: JaxArray, /, *, axis: Optional[int] = -1, out: Optional[JaxArray] = None
 ) -> JaxArray:
     if axis is None:
```

Comment:
Also, this line becomes redundant in that case, assuming we aren't accepting `None` as the `axis` from the user. Similar changes apply to all other functions

Reply:
That is correct, on it.